.envrc

```diff
@@ -5,6 +5,16 @@
 # If you'd like to override or set any custom environment variables, this .envrc will read a .env file at the end.
 set -e
 
+# Upgrading Mac can uninstall the Command Line Tools, thus, removing our access to git
+# The message talks about xcrun, however, we can use the lack of git as a way to know that we need this
+# xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools),
+# missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
+if [ "$(uname -s)" == "Darwin" ] && [ ! -f "/Library/Developer/CommandLineTools/usr/bin/git" ]; then
+    xcode-select --install
+    echo -e "$(tput setaf 1)\nERROR: Complete the interactive installation (10+ mins) of the Command Line Tools.$(tput sgr0)"
```

Comment:
Should we display them message before calling `xcode-select --install`? And if so, the message should reflect that we are about to install it

Reply:
It detaches but yes, you're right. The reader of the code can be confused by it.

Reply:
Re-ordered now.